components/mail/fetchmail/Makefile

```diff
@@ -22,54 +22,36 @@
 #
 # Copyright (c) 2011, 2013, Oracle and/or its affiliates. All rights reserved.
 # Copyright 2018, Michal Nowak
+# Copyright 2021, Nona Hansel
 #
 
-PREFERRED_BITS=64
-
+BUILD_BITS= 64
+USE_OPENSSL11= yes
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		fetchmail
-COMPONENT_VERSION=	6.3.26
-COMPONENT_REVISION=	3
+COMPONENT_VERSION=	6.4.19
+COMPONENT_DESCRIPTION=	'Full-featured, robust, well-documented remote-mail retrieval and forwarding utility'
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
```

Comment:
Don't put hyphens around the description text. In the manifest the text will be sourrounded by "".

Reply:
Removed them, thanks.